internal/promotion/simple_engine_test.go

```diff
@@ -601,6 +601,13 @@ func TestSimpleEngine_executeSteps(t *testing.T) {
 						Output: map[string]any{"key": "value"},
 					},
 				},
+				&promotion.MockStepRunner{
+					StepName: "skipped-step",
+					RunResult: promotion.StepResult{
+						Status: kargoapi.PromotionStepStatusSkipped,
+						Output: map[string]any{"key": "value"},
+					},
+				},
```

Comment:
You do not appear to have added any new tests cases that utilize this runner.

Reply:
added now. PTAL.